lib/chip/README.md

```diff
@@ -0,0 +1,24 @@
+# CHIP
+Chip stands for Check-in Integration Point
+
+## Description
+Provide Veterans with a unified front door experience for preparing for and checking into their clinical appointments. Including, changing the process for outpatient clinical workflow that improves efficiency and decrease devices requiring publicly accessible shared surfaces that may increase the transmissions of communicable diseases
+
+## Design
+
+### CHIP settings and setup
+CHIP settings are stored at `config/settings.local.yml` under `chip_api_v2`. Initialize your setup using a client (or custom built) session key
```

Comment:
We should mention that consumers of this lib will need to request a tenant id, username and password through our team. Also, not sure if we need to mention `settings.local.yml` because changes to that are usually not pushed to the repo: https://depo-platform-documentation.scrollhelp.site/developer-docs/settings#Settings-OverridingSettings